src/software/world/timestamped_robot_state.h

```diff
@@ -52,6 +52,14 @@ class TimestampedRobotState
      */
     RobotState robotState() const;
 
+
+    /**
+     * Returns a mutable reference to the robot state without any timestamp information.
+     *
+     * @return mutable robot state
+     */
+    RobotState &mutableRobotState();
```

Comment:
In order to be consistent with the rest of our repo (specifically looking at `World`), could we have `robotState` return a const reference here?

Reply:
changed to setRobotState